src/frontend-admin/src/components/App/App.tsx

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import { Route, NavLink, BrowserRouter as Router, Switch} from 'react-router-dom'; 
+import {HelloUser} from '../HelloUser/HelloUser'
 
 // Styles
 import styles from './App.module.css';
 
 // Primary Component
 export const App: React.FC = () => {
+  
   return (
     <div className={styles.app}>
       {/*header*/}
 
       <Router>
         <div className={styles.navContainer}>
+          <HelloUser name={"dan"}/> {/* Need to get name from contex? */}
```

Comment:
Yep, it'll come from context. Good place for a `// TODO Update with real name from context when available`